kas-installer.env.example

```diff
@@ -58,30 +65,20 @@ KAS_FLEET_MANAGER_IMAGE_REPOSITORY_PASSWORD=${KAS_FLEET_MANAGER_IMAGE_REPOSITORY
 # Default: None
 #RH_ORG_ID=<value>
 
-# [optional] Kafka producer/consumer quota limit
-# Default: 2Mi; performance=30Mi
-#KAFKA_CAPACITY_INGRESS_THROUGHPUT=<value>
-
-# [optional] Kafka max connections
-# Default: 100; performance=500
-#KAFKA_CAPACITY_TOTAL_MAX_CONNECTIONS=<value>
-
-# [optional] Kafka persistent volume capacity among the 3 of them
-# Default: 60Gi; performance=1000Gi
-#KAFKA_CAPACITY_MAX_DATA_RETENTION_SIZE=<value>
-
-# [optional] Kafka max partitions
-# Default: 100; performance=500
-#KAFKA_CAPACITY_MAX_PARTITIONS=<value>
-
-# [optional] Kafka data retention period
-# Default: P14D
-#KAFKA_CAPACITY_MAX_DATA_RETENTION_PERIOD=<value>
-    
-# [optional] Kafka connections attempts per second
-# Default: 100    
-#KAFKA_CAPACITY_MAX_CONNECTION_ATTEMPTS_PER_SEC=<value>
-
-# [optional] Admin Server - Use Edge Termination
-# Default: false
-#MANAGEDKAFKA_ADMINSERVER_EDGE_TLS_ENABLED=false
+# [optional] Fleet manager supported instance types (YAML)
+# Default: see kas-fleet-manager/templates/service-template.yml (parameter SUPPORTED_INSTANCE_TYPES)
+# SUPPORTED_INSTANCE_TYPES=<YAML value>
```

Comment:
@agullon - FYI, this may be used to provide custom values for instance capacity.